dune_client/models.py

```diff
@@ -236,6 +276,15 @@ def from_dict(cls, data: dict[str, RowData | MetaData]) -> ExecutionResult:
             metadata=ResultMetadata.from_dict(data["metadata"]),
         )
 
+    def __add__(self, other: ExecutionResult) -> ExecutionResult:
+        """
+        Enables combining results using the `+` operator.
+        """
+        self.rows += other.rows
```

Comment:
will this imply a copy of self.rows to make space for other.rows ?
or will it just grow the array of rows in place?